application-numbered-content-headings/application-numbered-content-headings-ui/src/main/resources/NumberedHeadings/Code/ActivationUI.xml

```diff
@@ -366,6 +503,7 @@ require(['jquery', 'NumberedHeadingsConfig', 'bootstrap'], function($, config) {
       <content>{{template name="display_macros.vm" /}}
 
 {{velocity}}
+#set ($discard = $xwiki.jsx.use('NumberedHeadings.Code.ActivationUI'))
```

Comment:
I'm not 100% sure but I think this is the wrong place as this is the code for the document information below the document and not the panel that is displayed next to the editor. The correct place should be the next object below.